app/Traits/ArclightConnection.php

```diff
@@ -31,8 +32,9 @@ private function fetchArclight($path, $language_id = null, $include_refs = false
             $results = json_decode(file_get_contents($path, false, $ctx));
         } catch (Exception $e) {
             return strpos($e, 'timed out') !== false
-                ? $this->setStatusCode(408)->replyWithError('Request timeout')
-                : $this->setStatusCode(500)->replyWithError('Internal server error');
+                ? $this->setStatusCode(HttpResponse::HTTP_REQUEST_TIMEOUT)->replyWithError('Request timeout')
+                : $this->setStatusCode(HttpResponse::HTTP_INTERNAL_SERVER_ERROR)
```

Comment:
see comment above. HTTP 409 should be returned

Reply:
@bradflood , I have updated the HTTP code DBP should return if arclight returns an error. You can see it in the following line:

https://github.com/faithcomesbyhearing/dbp/pull/867/files#diff-050f24b8934bfb79240c255c161a6789b43ed28255f2e543004fa8d47d8dad78R36